### ISSUE TITLE
How to hide notification when download complete

### ISSUE THREAD

**Issue by arifaizin** [tonyofrancis/Fetch#408]
[Question] I make many group download request.. so the notification really disturbing when download complete.. can i hide notification when download complete.. or just show one notification for all group request to show complete message.. and show retry if download  #failed? i have read the sample app but not find this

**Comment by tonyofrancis** [comment]
@arifaizin I am currently working on fixes a few things with the fetch notification manager. Once I complete the updates I will let you know.

**Comment by tonyofrancis** [comment]
This issue is now fixed in new Fetch version 3.0.10/Androidx 3.1.4. If you need more customization. You can try this code.
```java
    private Fetch fetch;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_download_list);
        setUpViews();
        final FetchConfiguration fetchConfiguration = new FetchConfiguration.Builder(this)
                .setDownloadConcurrentLimit(4)
                .setHttpDownloader(new OkHttpDownloader(Downloader.FileDownloaderType.PARALLEL))
                .setNamespace(FETCH_NAMESPACE)
                .setNotificationManager(new DefaultFetchNotificationManager(this) {
                    @NotNull
                    @Override
                    public Fetch getFetchInstanceForNamespace(@NotNull String namespace) {
                        return fetch;
                    }

                    @Override
                    public boolean shouldCancelNotification(@NotNull DownloadNotification downloadNotification) {
                        if (downloadNotification.getStatus() == Status.CANCELLED) {
                            return true;
                        } else {
                            return super.shouldCancelNotification(downloadNotification);
                        }
                    }
                })
                .build();
        fetch = Fetch.Impl.getInstance(fetchConfiguration);
    }
```
Note the `shouldCancelNotification` method.

**Comment by rahul5629** [comment]
sir when i click on pause in notification click then notification dispear
